cli/src/main/java/org/jboss/as/cli/impl/CommandContextImpl.java

```diff
@@ -1634,6 +1635,9 @@ private void handleCommand(ParsedCommandLine parsed) throws CommandFormatExcepti
                             invContext.getConfiguration().getOutputRedirection().close();
                         } catch (IOException ex) {
                             // Message must contain the Exception and the localized message.
+                            if (ex instanceof AccessDeniedException) {
+                                throw new CommandLineException(ex.getLocalizedMessage() + " (Permission denied)");
+                            }
                             throw new CommandLineException(ex.toString());
```

Comment:
Would not be attaching cause (at least for the else part) appropriate?

Reply:
@hkalina Thanks for taking a look. If part is requested simplification of the message this issue is about, the else part has not been changed, if you think that the cause is required, it should probably be covered by the different issue

Reply:
Attaching the cause has an impact on the content displayed to the user. We need to find the right balance of exposed content to a user (as opposed to a programmatic call that would require all full stacks).